lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -60,6 +61,28 @@ function getNodesAndTimingByUrl(nodeTimings) {
   return urlMap;
 }
 
+/**
+ * @param {Node} node
+ * @param {LH.Gatherer.Simulation.NodeTiming} nodeTiming
+ * @param {LH.Artifacts.DetectedStack[]} Stacks
+ */
+function computeStackSpecificTiming(node, nodeTiming, Stacks) {
+  const stackSpecificTiming = Object.assign({}, nodeTiming);
+  if (!Stacks) return stackSpecificTiming;
+  if (Stacks.some(stack => stack.id === 'amp')) {
+    // AMP will load a linked stylesheet asynchronously if it has not been loaded after 1 second [https://github.com/ampproject/amphtml/blob/master/src/font-stylesheet-timeout.js]
+    // Any potential savings for AMP stylesheet nodes must therefore be capped at 1 second.
+    // https://github.com/GoogleChrome/lighthouse/issues/2832#issuecomment-591066081
```

Comment:
let's move this comment line to the jsdoc description of the function explaining this is where any stack-specific overrides should happen